qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/GossipDifferentNodeJoinTest.java

```diff
@@ -1,132 +0,0 @@
-/*
- * Copyright © 2017 camunda services GmbH (info@camunda.com)
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.zeebe.broker.it.clustering;
-
-import static io.zeebe.broker.it.clustering.ClusteringRule.BROKER_1_CLIENT_ADDRESS;
-import static io.zeebe.broker.it.clustering.ClusteringRule.BROKER_1_TOML;
-import static io.zeebe.broker.it.clustering.ClusteringRule.BROKER_2_CLIENT_ADDRESS;
-import static io.zeebe.broker.it.clustering.ClusteringRule.BROKER_2_TOML;
-import static io.zeebe.broker.it.clustering.ClusteringRule.BROKER_4_CLIENT_ADDRESS;
-import static io.zeebe.broker.it.clustering.ClusteringRule.BROKER_4_TOML;
-import static io.zeebe.protocol.Protocol.DEFAULT_TOPIC;
```

Comment:
Why this test was deleted? I can remember that gossip joining with different nodes does not work in the past :sweat_smile:

Reply:
I didn't come up with a good solution to configure a different initial contact point while using dynamic port assignments. And I hope that the gossip test already covers the issues you mentioned https://github.com/zeebe-io/zeebe/blob/1150-parallel-test-execution/gossip/src/test/java/io/zeebe/gossip/GossipJoinTest.java#L190.

Reply:
Perhaps choosing, in the ClusteringRule, randomly the initial points or brokerId -1. So Node 1 connects to 0 and Node 2 to 1.

Reply:
> So Node 1 connects to 0 and Node 2 to 1.

I can do that if you want but then we still don't need the test case cause it is identical with `GossipClusteringTest.java` if I remember correctly.